PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/ui/WebAppSettingPanel.java

```diff
@@ -1,876 +0,0 @@
-import org.jetbrains.idea.maven.model.MavenConstants;
-import org.jetbrains.idea.maven.project.MavenProject;
-
-import com.intellij.icons.AllIcons;
-import com.intellij.openapi.actionSystem.ActionToolbarPosition;
-import com.intellij.openapi.actionSystem.AnActionEvent;
-import com.intellij.openapi.project.Project;
-import com.intellij.openapi.util.Comparing;
-import com.intellij.packaging.artifacts.Artifact;
-import com.intellij.ui.AnActionButton;
-import com.intellij.ui.HideableDecorator;
-import com.intellij.ui.HyperlinkLabel;
-import com.intellij.ui.ListCellRendererWrapper;
-import com.intellij.ui.ToolbarDecorator;
-import com.intellij.ui.table.JBTable;
-import com.microsoft.azure.management.appservice.AppServicePlan;
-import com.microsoft.azure.management.appservice.DeploymentSlot;
-import com.microsoft.azure.management.appservice.OperatingSystem;
-import com.microsoft.azure.management.appservice.PricingTier;
-import com.microsoft.azure.management.appservice.RuntimeStack;
-import com.microsoft.azure.management.appservice.WebApp;
-import com.microsoft.azure.management.resources.Location;
-import com.microsoft.azure.management.resources.ResourceGroup;
-import com.microsoft.azure.management.resources.Subscription;
-import com.microsoft.azuretools.authmanage.AuthMethodManager;
-import com.microsoft.azuretools.core.mvp.model.ResourceEx;
-import com.microsoft.azuretools.core.mvp.model.webapp.JdkModel;
-import com.microsoft.azuretools.utils.WebAppUtils;
-import com.microsoft.intellij.runner.AzureSettingPanel;
-import com.microsoft.intellij.runner.webapp.Constants;
-import com.microsoft.intellij.runner.webapp.webappconfig.WebAppConfiguration;
-import com.microsoft.intellij.util.MavenRunTaskUtil;
-
-import icons.MavenIcons;
-
-public class WebAppSettingPanel extends AzureSettingPanel<WebAppConfiguration> implements WebAppDeployMvpView {
-
-    // const
-    private static final String RESOURCE_GROUP = "Resource Group";
-    private static final String APP_SERVICE_PLAN = "App Service Plan";
-    private static final String RUNTIME = "Runtime";
-    private static final String DEPLOYMENT_SLOT = "Deployment Slot";
-    private static final String NOT_APPLICABLE = "N/A";
-    private static final String TABLE_LOADING_MESSAGE = "Loading ... ";
-    private static final String TABLE_EMPTY_MESSAGE = "No available Web App.";
-    private static final String DEFAULT_APP_NAME = "webapp-";
-    private static final String DEFAULT_SLOT_NAME = "slot-";
-    private static final String DEFAULT_PLAN_NAME = "appsp-";
-    private static final String DEFAULT_RGP_NAME = "rg-webapp-";
-    private static final String WEB_CONFIG_URL_FORMAT = "https://%s/dev/wwwroot/web.config";
-    private static final String NON_WAR_FILE_DEPLOY_HINT = "Please check the web.config file used to deploy this" +
-            " non-WAR executable.";
-    // presenter
-    private final WebAppDeployViewPresenter<WebAppSettingPanel> webAppDeployViewPresenter;
```

Comment:
class `WebAppDeployViewPresenter` could also be removed

Reply:
Done